pvlib/inverter.py

```diff
@@ -348,8 +348,9 @@ def pvwatts(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
 
     Parameters
     ----------
-    pdc: numeric
-        DC power. Same unit as ``pdc0``.
+    p_dc : tuple, list or array of numeric
+        DC power on each MPPT input of the inverter. If type is array, must
+        be 2d with axis 0 being the MPPT inputs. Same unit as ``pdc0``.
```

Comment:
This makes sense for `pvwatts_multi`, but is it also true here in `pvwatts`?

Reply:
The perils of cut and paste

Reply:
Oh, `p_dc` vs `pdc` discrepancy between docstring and signature for both functions